src/javascript/javascript-parser.ts

```diff
@@ -27,6 +27,16 @@ declare class SyntaxError {
   column: number;
 }
 
+export interface Comment {
+  type: string;
+  value: string;
+  loc: SourceLocation;
+}
+
+export interface Program extends EstreeProgram {
+  comments: Comment[];
```

Comment:
> Were comments already being attached to Programs and this is just to satisfy the type checker?

Yep! `comments` is added by `espree.parse` via the `comment` option: https://github.com/Polymer/polymer-analyzer/blob/master/src/javascript/javascript-parser.ts#L48